src/test/java/org/apache/commons/compress/archivers/tar/SparseFilesTest.java

```diff
@@ -73,54 +75,57 @@ public void testPaxGNU() throws Throwable {
 
     @Test
     public void testExtractSparseTarsOnWindows() throws IOException {
-        if (!isOnWindows) {
-            return;
-        }
+        assumeTrue(isOnWindows);
```

Comment:
Maybe we can make it more detailed with something like :
```
assumeTrue("This test should be ignored if not running on Windows",isOnWindows);
```

Reply:
Good idea. I added the messages everywhere